lib/reline/key_actor/composite.rb

```diff
@@ -0,0 +1,17 @@
+class Reline::KeyActor::Composite
+  def initialize(key_actors)
+    @key_actors = key_actors
+  end
+
+  def matching?(key)
+    @key_actors.any? { |key_actor| key_actor.matching?(key) }
+  end
+
+  def get(key)
+    @key_actors.each do |key_actor|
+      func = key_actor.get(key)
+      return func if func
+    end
+    nil
```

Comment:
```suggestion
    @key_actors.find { |key_actor| key_actor.get(key) }
```

Reply:
Using find need to call `get(key)` two times.
```ruby
@key_actors.find { |key_actor| key_actor.get(key) }.get(key)
```
comparing the two code, IMO both are fine.

Reply:
Ah I missed that. Let's keep it as it is then.